src/content/en/updates/2020/05/devtools.md

```diff
@@ -0,0 +1,177 @@
+Hover over the **Moved from** and **Moved to** fields to visualize where the layout shift
+occurred.
+
+![The details of a layout shift.](/web/updates/images/2020/05/cls.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f130314cc50ccd2c933017ee9bb06eefd68ed206 #}
+
+## More accurate promise terminology in the Console {: #promises }
+
+When logging a `Promise` the Console used to incorrectly describe the state of the `Promise`
+as `resolved`:
+
+![An example of the Console using the old "resolved" terminology.](/web/updates/images/2020/05/resolved.jpg)
+
+The Console now uses the term `fulfilled`, which [aligns with the `Promise` spec](https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md):
+
+![An example of the Console using the new "fulfilled" terminology.](/web/updates/images/2020/05/fulfilled.jpg)
+
+{# https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/2141673 #}
+
+V8 Bug: [#6751](https://bugs.chromium.org/p/v8/issues/detail?id=6751)
+
+## Styles pane updates {: #styles }
+
+### Support for the `revert` keyword {: #revert }
+
+[revert]: https://developer.mozilla.org/en-US/docs/Web/CSS/revert
+[revert-bcd]: https://developer.mozilla.org/en-US/docs/Web/CSS/revert#Browser_compatibility
+
+The Styles pane's autocomplete UI now detects the [`revert`][revert] CSS keyword, which
+reverts the cascaded value of a property to what the value would have been if no changes
+had been made to the element's styling.
+
+![Setting the value of a property to revert.](/web/updates/images/2020/05/revert.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/811c6e98ee25baf3f6f5a6145e1896fbe4362d7a #}
+
+Chromium Bug: [#1075437](https://crbug.com/1075437)
+
+### Image previews {: #image-previews }
+
+Hover over a `background-image` value in the Styles pane to see a preview of
+the image in a tooltip.
+
+![Hovering over a background-image value.](/web/updates/images/2020/05/image-preview.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/1ebbb30be21f07d6a4d3ccd3b8dc7bd0dd37b41d #}
+
+Chromium Bug: [#1040019](https://crbug.com/1040019)
+
+### Color Picker now uses space-separated functional color notation {: #color }
+
+[CSS Color Module Level 4](https://drafts.csswg.org/css-color/#changes-from-3)
+specifies that color functions like `rgb()` should support space-separated
+arguments. For example, `rgb(0, 0, 0)` is equivalent to `rbg(0 0 0)`.
+
+When you choose colors with the [Color Picker](/web/tools/chrome-devtools/css/reference#color-picker)
+or alternate between color representations in the Styles pane by holding <kbd>Shift</kbd> and then
+clicking the color value, you'll now see the space-separated argument syntax.
+
+![Using space-separated arguments in the Styles pane.](/web/updates/images/2020/05/color.jpg)
+
+DevTools is using the new syntax because [upcoming CSS features like `color()` do not
```

Comment:
```suggestion
![Using space-separated arguments in the Styles pane.](/web/updates/images/2020/05/color.jpg)

You'll also see the syntax in the Computed pane and the Inspect Mode tooltip. 

DevTools is using the new syntax because [upcoming CSS features like `color()` do not
```